data/mods/BlazeIndustries/vehicles/vehicles.json

```diff
@@ -0,0 +1,186 @@
+      { "x": 2, "y": 2, "part": "headlight_reinforced" },
+      { "x": 2, "y": 3, "parts": [ "hdframe_ne", "hdhalfboard_ne", "plating_military" ] },
+      { "x": 2, "y": 3, "parts": [ "wheel_mount_heavy_steerable", "wheel_armor" ] },
+      { "x": 2, "y": 0, "parts": [ "hdframe_horizontal", "hdhalfboard_horizontal" ] },
+      { "x": 2, "y": 0, "parts": [ "plating_military", "headlight_reinforced" ] },
+      { "x": 2, "y": -1, "parts": [ "hdframe_nw", "hdhalfboard_nw", "plating_military" ] },
+      { "x": 2, "y": -1, "parts": [ "wheel_mount_heavy_steerable", "wheel_armor" ] },
+      { "x": 3, "y": 1, "parts": [ "hdframe_horizontal_front", "plating_military" ] },
+      { "x": 3, "y": 2, "parts": [ "hdframe_horizontal_front", "plating_military" ] },
+      { "x": 3, "y": 3, "parts": [ "hdframe_ne", "plating_military" ] },
+      { "x": 3, "y": 0, "parts": [ "hdframe_horizontal_front", "plating_military" ] },
+      { "x": 3, "y": -1, "parts": [ "hdframe_nw", "plating_military" ] },
+      { "x": -1, "y": 1, "parts": [ "hdframe_horizontal_2", "aisle_horizontal", "roof", "turret_mount" ] },
+      {
+        "x": -1,
+        "y": 1,
+        "part": "mounted_mk19",
+        "ammo": 60,
+        "ammo_types": [ "40x46mm_m433", "40mm_emp" ],
+        "ammo_qty": [ 1, 25 ]
+      },
+      { "x": -1, "y": 2, "parts": [ "hdframe_horizontal_2", "seat", "seatbelt", "hdroof" ] },
+      { "x": -1, "y": 3, "parts": [ "hdframe_vertical", "hddoor_right" ] },
+      { "x": -1, "y": 0, "parts": [ "hdframe_horizontal_2", "seat", "seatbelt", "hdroof" ] },
+      { "x": -1, "y": -1, "parts": [ "hdframe_vertical", "hddoor_left" ] },
+      { "x": -2, "y": 1, "parts": [ "hdframe_horizontal_2", "trunk", "hdroof" ] },
+      { "x": -2, "y": 2, "parts": [ "hdframe_horizontal_2", "trunk", "hdroof" ] },
+      { "x": -2, "y": 3, "parts": [ "hdframe_vertical", "hdboard_vertical_right", "plating_military" ] },
+      { "x": -2, "y": 0, "parts": [ "hdframe_horizontal_2", "trunk", "hdroof" ] },
+      { "x": -2, "y": -1, "parts": [ "hdframe_vertical", "hdboard_vertical_left", "plating_military" ] },
+      { "x": -3, "y": 1, "parts": [ "hdframe_horizontal_2", "hddoor_trunk", "plating_military" ] },
+      { "x": -3, "y": 2, "parts": [ "hdframe_horizontal_2", "hddoor_trunk", "plating_military" ] },
+      { "x": -3, "y": 3, "parts": [ "hdframe_se", "hdboard_se", "plating_military" ] },
+      { "x": -3, "y": 3, "parts": [ "wheel_mount_heavy", "wheel_armor" ] },
+      { "x": -3, "y": 0, "parts": [ "hdframe_horizontal", "hddoor_trunk", "muffler" ] },
+      { "x": -3, "y": 0, "part": "plating_military" },
+      { "x": -3, "y": -1, "parts": [ "hdframe_sw", "hdboard_sw", "plating_military" ] },
+      { "x": -3, "y": -1, "parts": [ "wheel_mount_heavy", "wheel_armor" ] }
+    ],
+    "items": [
+      { "x": 0, "y": 0, "chance": 5, "items": [ "id_military" ] },
+      { "x": -2, "y": 0, "chance": 5, "item_groups": [ "fuel_diesel" ] }
+    ]
+  }
```

Comment:
I'm not seeing the grenade launcher APC in this file, only the humvee?

Reply:
Nevermind I think that vehicle is already gone.

Reply:
I didn't know there was a grenade launcher apc in aftershock. Is it vanilla?

Reply:
Oh, I think I see the one you mean. Is "aapc-gl" the one you meant?